main/src/ca/uwaterloo/flix/util/Formatter.scala

```diff
@@ -26,25 +70,17 @@ trait Formatter {
 
   def bold(s: String): String
 
-  def underline(s: String): String
+  def underline(s: String): String = Console.UNDERLINED + s + Console.RESET
```

Comment:
You are right. I think we should only underline the color formatter. I will let you fix it in a new PR.